CI/unit_tests/utils/test_scaling_functions.py

```diff
@@ -0,0 +1,131 @@
+"""
+MDSuite: A Zincwarecode package.
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+SPDX-License-Identifier: EPL-2.0
+Copyright Contributors to the Zincwarecode Project.
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+Unit tests for the scaling functions module.
+"""
+import unittest
+
+import numpy as np
+
+from mdsuite.utils.scale_functions import (
+    linear_scale_function,
+    linearithmic_scale_function,
+    polynomial_scale_function,
+    quadratic_scale_function,
+)
+
+
+class TestScalingFunctions(unittest.TestCase):
+    """
+    A test class for the scaling functions module.
+    """
+
+    def test_linear_scaling_function(self):
+        """
```

Comment:
I think for all the scaling tests it would be better to test them against a function that actually scales like this. 
Currently you are only testing that the numpy code you wrote works. This ensures that no one accidentally changes the code but does not ensure that the scaling actually is the scaling we are interested in.

Reply:
I compute the value by hand as it is a simple algebraic expression. The only time I did not do this was for the log one. I didn't use the output of the function as the test comparison I wrote the function out, computed it on the values and then tested that those numbers came out of the tests. We could write it such that it is tested on random numbers and then use a written polynomial for each but I think this suffices.